README.md

```diff
@@ -182,7 +182,6 @@ SIG-specific GitHub discussions.
 
 | Name | Meeting Time | Meeting Notes | Slack Channel | Meeting Invites Group | [Governance Committee](./community-members.md#governance-committee) Liaison |
 |------|--------------|---------------|---------------|-----------------|--------------------------------|
-| <a id="sig-maintainers"></a>Maintainer Meeting | Monday at 09:00 PT | [Google Doc](https://docs.google.com/document/d/1_X2CTiDXmxk2JozBNwQ2uP8FfV1rxx9ISE5puHu0yIk) | [#otel-maintainers](https://cloud-native.slack.com/archives/C01NJ7V1KRC) | [calendar-maintainer-meeting](https://groups.google.com/a/opentelemetry.io/g/calendar-maintainer-meeting) | [Morgan McLean](https://github.com/mtwo) |
```

Comment:
You need to edit the YAML file that defines this table too

Reply:
Aha, on it!

Reply:
Looks like @arminru added this to my PR, thanks Armin!